plotly/tools.py

```diff
@@ -1443,6 +1443,376 @@ class FigureFactory(object):
     """
 
     @staticmethod
+    def create_scatterplotmatrix(df, useindex = False, index = " ",
+                                 diagonal = "Scatter", symbol = 0, size = 6,
+                                 height = " ", width = " ", jitter = 0,
+                                 title = "Scatterplot Matrix"):
```

Comment:
We typically keep pretty lengthy docs for [things like this](https://github.com/plotly/plotly.py/blob/master/plotly/tools.py#L1663-LL1742)

This way, when users run `help`, the information is right there.